java/client/src/org/openqa/selenium/remote/DriverCommand.java

```diff
@@ -227,6 +228,11 @@ static CommandPayload SET_ALERT_VALUE(String keysToSend) {
 
   String SET_TIMEOUT = "setTimeout";
 
+  String PRINT_PAGE = "printPage";
+  static CommandPayload PRINT_PAGE(PrintOptions options) {
+    return new CommandPayload(PRINT_PAGE, options.toJson());
```

Comment:
Ah! This explains why you're exposing the json serialisation. Please follow the pattern used by the rest of this class.